docsrc/source/pages/use_cases/comparing_datasets.rst

```diff
@@ -0,0 +1,52 @@
+==================
+Dataset Comparison
+==================
+
+*This feature was introduced in pandas-profiling 3.4.*
```

Comment:
Please remove this reference from the documentation. 

We can add it later with a changelog reference for instance.